root/edit/search_macros.tt

```diff
@@ -4,6 +4,12 @@
 </span>
 [% END %]
 
+[% BLOCK search_heading %]
+  <h1>[% l('Search for Edits') %]</h1>
+  <p>[% l('The filters below allow you to find only edits fulfilling specific criteria. See the {doc_link|documentation} for more info about using the filters.', {doc_link => doc_link('How to Use the Edit Search')}) %]</p>
+  <p>[% l('If you want to save an edit search for future use, you can just bookmark it!') %]</p>
```

Comment:
That would probably be more helpful to provide an actual button to bookmark it or at least to copy/share the URL.

Reply:
Don't most browsers just have a trivial way to do that from the page itself, which would be easier than copying the URL?

Reply:
Bookmarking seems _trivial_ to me indeed, so it did not seem worth mentioning it. Having a button to copy the URL to share with the community forum seem more useful, but that doesn’t seem _trivial_ to implement, so probably not worth it either.